e2e/sites.test.ts

```diff
@@ -8,81 +7,106 @@ import MainSidebar from './page-objects/main-sidebar';
 import Onboarding from './page-objects/onboarding';
 import SiteContent from './page-objects/site-content';
 import WhatsNewModal from './page-objects/whats-new-modal';
+import fs from 'fs-extra';
 import { getUrlWithAutoLogin } from './utils';
 
 const skipTestOnWindows = process.platform === 'win32' ? test.skip : test;
+const DEFAULT_SITE_NAME = 'My WordPress Website';
```

Comment:
We use it in other e2e tests. Does it make sense to extract to a constants file at this point?